config/percep/config.json

```diff
@@ -0,0 +1,58 @@
+{
+    "frame_write_interval": 10,
+    "buffer_iterations": 20,
+    "DEFAULT_TAG_VAL": -1,
+    "mm_per_m":1000,
+    "threshold_confidence":90,
```

Comment:
NIT: these items seem to have a specific use but are not grouped together like other items, why is that?